ui/app/components/job-status/panel/deploying.js

```diff
@@ -61,7 +61,7 @@ export default class JobStatusPanelDeployingComponent extends Component {
   fail;
 
   @alias('job.latestDeployment') deployment;
-  @alias('deployment.desiredTotal') desiredTotal;
+  @alias('totalAllocs') desiredTotal;
```

Comment:
This was a change to address a bug found when restarting a dead job. This bug is present in Nomad's UI today. The latest deployment on a job that you start from dead have a desired count that differs from the sum of the job's groups.count, which is the number the operator is really going for.